### PR TITLE
jasmine-rails - ready

### PR DESCRIPTION
**Pull request by brouhaha88**
- [x] @vovka 
- [x] @AntonMayer 
- [x] @mrJmek

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  mount JasmineRails::Engine => '/specs' if defined?(JasmineRails)
```
Comment (brouhaha88) [inline config/routes.rb]: @vovka

**Comment by mrJmek** [comment]
git pull origin master
git push origin jasmine/setup_jasmine

without git add and without commit - just to resolve last commit 'Update README.md'

**Comment by vovka** [comment]
Looks good. Can be merged.

**Comment by brouhaha88** [comment]
Jira task # LR-147

**Pull request merged**